.terraform-docs.yml

```
@@ -41,7 +41,12 @@ content: |-
 
   {{ indent 0 "#" }} Usage
 
-  It's very easy to use!
+  To use this module a resource group and log analytics workspace is required. 
+  The webhook URL needs to point to a valid receiver for pipeline events.
+  If authentication or other options are required they need to be included in the URL as path or query parameters.
+
+  Optionally you can specify additional kusto queries to monitor. See `examples/extra_queries/main.tf` for details.
+  
   ```hcl
   {{ include "examples/basic/main.tf" }}
   ```
```

```suggestion
  To use this module a resource group and log analytics workspace is required. 
  The webhook URL needs to point to a valid receiver for pipeline events.
  If authentication or other options are required they need to be included in the URL as path or query parameters.
  
  ```hcl
  {{ include "examples/basic/main.tf" }}
  ```

{{ indent 1 "#" }} Extra Queries

You can specify additional kusto queries to monitor. 
  ```hcl
  {{ include "examples/extra_queries/main.tf" }}
  ```
`queries/failed_jobs.kusto`
  ```kusto
  {{ include "examples/extra_queries/queries/failed_jobs.kusto" }}
  ```
```